catalog/search-request.schema.json

```diff
@@ -85,77 +84,77 @@
       }}
     }},
     "facets": { "type": "array", "uniqueItems": true, "items": { "enum": [
-      "offerType",
-      "firstCirculationYear",
+      "offer_type",
+      "first_circulation_year",
       "mileage.km.slice",
       "euronorm",
       "critair",
-      "sellingPriceWithTaxes.slice",
-      "sellingPriceWithTakeOverWithTaxes.slice",
-      "indicativeLoadInstallment.slice",
+      "selling_price_with_taxes.slice",
+      "selling_price_with_takeover_with_taxes.slice",
+      "indicative_load_installment.slice",
       "maker",
       "model",
-      "simpleColor",
-      "simpleEquipment",
+      "simple_color",
+      "simple_equipment",
       "category",
       "segment",
-      "energyType",
+      "energy_type",
       "transmission",
       "drive",
       "promotion",
       "doors",
       "seats"
     ]}},
     "order": { "type": "array", "uniqueItems": true, "items": {"type": "object", "additionalProperties": false, "minProperties":1, "maxProperties": 1, "properties": {
-      "publicationDate": { "enum": ["asc", "desc"] },
+      "publication_date": { "enum": ["asc", "desc"] },
       "mileage.km": { "enum": ["asc", "desc"] },
-      "sellingPriceWithTaxes": { "enum": ["asc", "desc"] },
-      "sellingPriceWithTakeOverWithTaxes": { "enum": ["asc", "desc"] },
-      "indicativeMinimumMonthlyInstallment": { "enum": ["asc", "desc"] },
-      "indicativeMinimumMonthlyInstallmentWithTakeover": { "enum": ["asc", "desc"] },
+      "selling_price_with_taxes": { "enum": ["asc", "desc"] },
+      "selling_price_with_takeover_with_taxes": { "enum": ["asc", "desc"] },
+      "indicative_minimum_monthly_installment": { "enum": ["asc", "desc"] },
+      "indicative_minimum_monthly_installment_with_takeover": { "enum": ["asc", "desc"] },
       "maker": { "enum": ["asc", "desc"] },
       "model": { "enum": ["asc", "desc"] },
       "power.ch": { "enum": ["asc", "desc"] },
-      "combinedCycleConsumption.ltPer100Km": { "enum": ["asc", "desc"] },
-      "combinedCycleConsumption.kwhPer100Km": { "enum": ["asc", "desc"] }
+      "combined_cycle_consumption.lt_per_100_km": { "enum": ["asc", "desc"] },
+      "combined_cycle_consumption.kwh_per_100_km": { "enum": ["asc", "desc"] }
     }}},
-    "limit": { "type": "integer", "minimum": 0, "default": 100 },
-    "offset": { "type": "integer", "minimum": 0, "default": 0 },
+    "page_size": { "type": "integer", "minimum": 0, "default": 100 },
+    "current_page": { "type": "integer", "minimum": 0, "default": 0 },
```

Comment:
Hmm. `current_page` has still `0` as `minimum` and `default`, so it is still `0-based`. However, we say "the first page" or "the second page", so intuitively we expect it to be `1-based`.